javascript/ql/src/semmle/javascript/dataflow/Configuration.qll

```diff
@@ -990,7 +1117,7 @@ private predicate reachableFromStoreBaseStep(
  *
  * In other words, `pred` may flow to `succ` through a property.
  */
-pragma[noinline]
+pragma[noopt]
```

Comment:
Have you tried to get around the performance issue every other way?

Reply:
_Every_ other way? Certainly not. But doing the gymnastics didn't seem worth it until we're sure this is going to land.